posthog/api/async_migration.py

```diff
@@ -78,6 +83,19 @@ def trigger(self, request, **kwargs):
         trigger_migration(migration_instance)
         return response.Response({"success": True}, status=200)
 
+    @action(methods=["POST"], detail=True)
+    def resume(self, request, **kwargs):
+        migration_instance = self.get_object()
+        if migration_instance.status != MigrationStatus.Errored:
+            return response.Response(
+                {"success": False, "error": "Can't resume a migration that isn't in errored state",}, status=400,
+            )
+        resumable, error = is_migration_resumable(migration_instance)
+        if not resumable:
+            return response.Response({"success": False, "error": error,}, status=500,)
```

Comment:
Let's not throw 500s. 

A 400 maybe? It is a bad request in the sense that someone called for it to be resumed but it can't be

Reply:
you and jams have opposite opinions here https://github.com/PostHog/posthog/pull/7885#discussion_r778398014 :D  but sure 400 sounds fine with me.